client/api/url.js

```diff
@@ -18,8 +23,10 @@ export const labelTestPrint = () => `${ namespace }label/preview`;
 
 export const addressNormalization = () => `${ namespace }normalize-address`;
 
-export const shippingServiceSettings = ( methodId, instanceId = false ) => instanceId
-	? `${ namespace }services/${ methodId }/${ instanceId }`
-	: `${ namespace }services/${ methodId }`;
+export const settingsForm = ( methodId, instanceId = false, formType = false ) => {
```

Comment:
Having some arguments have default values it's confusing, since this function is always called with the 3 arguments. Also, I think `formType` should be the first, since it'll end up being the first chunk of the URL (will make the calling code a bit clearer maybe).

Reply:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1171/commits/7248ece5d72e1b0228398e34a213f569d5e483e9.